source/upgrade/important-upgrade-notes.rst

```diff
@@ -2,14 +2,50 @@ Important Upgrade Notes
+|                                                    |                                                                                                                                                                  |
+|                                                    | 2. Medium effort, medium downtime - This process will require SQL queries to be executed manually on the server. To avoid causing a table lock, a customer can   |
+|                                                    | choose to use the pt-online-schema-change tool for MySQL. For Postgres, the table locking is very minimal. The advantage is that since there are a lot of        |
+|                                                    | queries, the customer can take their own time to run individual queries during off-hours. All queries except #11 are safe to be executed this way. Then the      |
+|                                                    | usual method of (1) can be followed.                                                                                                                             |
+|                                                    |                                                                                                                                                                  |
+|                                                    | 3. High effort, low downtime - This process requires everything of (2), plus it tries to minimize the impact of query #11. We can do this by following step 2,   |
+|                                                    | and then starting v6 along with a running v5 server, and then monitor the application logs. As soon as the v6 application starts up, we need to bring down a v5  |
+|                                                    | node. This minimizes the downtime to only a few seconds.                                                                                                         |
+|                                                    |                                                                                                                                                                  |
+|                                                    | It is recommended to start Mattermost directly and not by system control to avoid interfering because system control thinks the Mattermost service hangs.        |
+|                                                    | If going with defaults, then the startup timeout should be 1 hour. If the migration is expected to take more than that then running the mattermost binary        |
+|                                                    | directly should avoid a possible kill in the middle of it.                                                                                                       |
```

Comment:
I'd simplify as follows:

> It is recommended to start Mattermost directly and not through systemctl to avoid the server process getting killed during the migration. This can happen since the value of `TimeoutStartSec` in the provided systemctl service file is set to 1 hour.